lib/Conversion/StandardToHandshake/StandardToHandshake.cpp

```diff
@@ -39,6 +39,7 @@ using namespace std;
 
 typedef DenseMap<Block *, vector<Value>> BlockValues;
 typedef DenseMap<Block *, vector<Operation *>> BlockOps;
+typedef DenseMap<Value, Operation *> blockArgPairs;
```

Comment:
Style thing: I think it should be `BlockArgPairs`. Otherwise the change makes sense.